supports/integration-test/reset_env.py

```diff
@@ -73,6 +73,27 @@ def test_create_10M_DFSIO(self):
             subprocess.call("hdfs dfs -mv /benchmarks/TestDFSIO/io_data " +
                             TEST_DIR + str(i) + "_data")
 
+
+    def test_create_10K_0KB_DFSIO_parallel(self):
+        dir_name = TEST_DIR + random_string()
+
+        """
+        When there are so many thread is started, and the client cannot handles them,
+        please set process_group_size less
+        """
+        process_group_size = 200
+
+        file_index = 0;
+        for i in range(10000 / process_group_size):
+            process_group = []
+            for j in range(process_group_size):
+                process_group.append(subprocess.Popen("hdfs dfs -touchz dir_name/" + str(file_index)))
+                file_index = file_index + 1
+
+            # wait
+            for k in range(process_group_size):
```

Comment:
You can refine this method based on [
SIMPLE WAY TO EXECUTE MULTIPLE PROCESS IN PARALLEL (PYTHON RECIPE)](http://code.activestate.com/recipes/577376-simple-way-to-execute-multiple-process-in-parallel/)